src/components/snackbar/snackbar.tsx

```diff
@@ -37,9 +43,7 @@ import translate from '../../global/translations';
  * @exampleComponent limel-example-snackbar-dismissible
  * @exampleComponent limel-example-snackbar-with-action
  * @exampleComponent limel-example-snackbar-with-changing-messages
- * @exampleComponent limel-example-snackbar-positioning
  */
-
 @Component({
```

Comment:
can't comment on the relevant prop, but I just realized that `public multiline: boolean;` doesn't really do anything. I don't think it ever did, even in the MDC version. What do you say? Remove it?

Reply:
oh, had a look at this old PR which was never merged: [`1ee660b` (#2403)](https://github.com/Lundalogik/lime-elements/pull/2403/commits/1ee660b1b2c6c66021b3b9b8e0584e8baab36c97) and saw that I added this description for the prop:
```md
     * When `true`, the layout of the Snackbar will change.
     * The action and the dismiss buttons will be rendered in a
     * row below the text, instead of being rendered as trailing elements.
     * This might be useful for Snackbars with very long messages.
     ```

Maybe this was something we got out of the box from MDC? I think we should update the new styles to accommodate for the `multiline` prop. This will be helpful when we allow having multiple actions in the snackbar too.

Reply:
I think I remember that `multiline` was a property on the MDC snackbar, but maybe they have removed it in later versions? :thinking:

Reply:
It's weird to keep it, as we can achieve the same results dynamically with CSS only to be honest. I think we can handle this in another PR, get rid of this prop, and make a breaking change or something

Reply:
Seems it was only used to set this class

![image](https://github.com/user-attachments/assets/bdc21a03-0ecd-4fe3-b376-4cde9cf1c502)

Reply:
Let's mark it as deprecated.

Reply:
c9babb7f4